docs/content/platform/blocks/firecrawl/crawl.md

```
@@ -0,0 +1,43 @@
+_Add technical explanation here._
+<!-- END MANUAL -->
+
+### Inputs
+| Input | Description | Type | Required |
+|-------|-------------|------|----------|
+| url | The URL to crawl | str | Yes |
+| limit | The number of pages to crawl | int | No |
+| only_main_content | Only return the main content of the page excluding headers, navs, footers, etc. | bool | No |
+| max_age | The maximum age of the page in milliseconds - default is 1 hour | int | No |
+| wait_for | Specify a delay in milliseconds before fetching the content, allowing the page sufficient time to load. | int | No |
+| formats | The format of the crawl | List["markdown" | "html" | "rawHtml"] | No |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)